tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "lib": ["es2015"],
+    "strict": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+
+    // this looks super weird, but baseUrl doesn't consider .
+    // a module so we have to explicitly tell it in paths...
```

Comment:
It's not too weird, many definitely typed repos have this configuration.
and most unified ecosystem repos do.